clover-core-libs/clover-slf4j/pom.xml

```diff
@@ -2,8 +2,8 @@
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>clover-log4j</artifactId>
-    <version>1.2.17</version>
+    <artifactId>clover-slf4j</artifactId>
+    <version>1.7.36</version>
```

Comment:
I wasn't possible to upgrade to the latest SLF4J 2.0.0 at the moment. 

A reasons for this is that we need to repack a JAR and add 'clover.' prefix to all packages and the JARJAR utility used for this wasn't able to parse the SLF4J 2.0 JAR file. Probably due to bytecode version used.

An upgrade to SLF4J 2.0.0 shall be possible once we replace JARJAR with ProGuard, see issue #102